gui/default/assets/lang/lang-en.json

```diff
@@ -374,11 +388,15 @@
    "You have no ignored folders.": "You have no ignored folders.",
    "You have unsaved changes. Do you really want to discard them?": "You have unsaved changes. Do you really want to discard them?",
    "You must keep at least one version.": "You must keep at least one version.",
+   "You should never add or change anything locally in a \"{%receiveEncrypted%}\" folder.": "You should never add or change anything locally in a \"{{receiveEncrypted}}\" folder.",
    "days": "days",
    "directories": "directories",
+   "enter password to encrypt sent data": "enter password to encrypt sent data",
```

Comment:
it is impossible to encrypt "sent" data. Perhaps "data to be sent", or just "enter password for data encryption"?